src/Assembly.js

```diff
@@ -165,6 +179,17 @@ class Assembly extends React.Component {
       )
     )
 
+    autorun(() => {
+      this.menu.name = this.participant_account.information.name
+      this.menu.phone_number = this.participant_account.information.phone_number
+      this.menu.treatment_start = this.participant_account.information.treatment_start
+    })
+
+    autorun(() => {
+      this.coordinator_menu.name = this.coordinator_account.information.name
+      this.coordinator_menu.email = this.coordinator_account.information.email
+    })
```

Comment:
Whenever we receive new account information from the server,
it overwrites our local copy.

It would be a big surprise if this ends up interfering substantially with peoples' experience;
there aren't many cases where the data will change on the server.